### PR TITLE
Fix yet another emulator precision indexing bug

### PR DESCRIPTION
**Pull request by dmcdougall**
Although this could have been avoided, please merge #631 before merging this PR.

### CODE REVIEW

File: src/gp/src/GPMSA.C [hunk 1]
```diff
@@ -401,7 +401,7 @@ GPMSAEmulator<V, M>::lnValue(const V & domainVector,
           covMatrix(i,j) += BT_Wy_B_inv(i,j) * inv_lambda_y;
 
       const double emulator_precision =
-        domainVector[dimParameter+1];
+        domainVector[dimParameter+(num_svd_terms<num_nonzero_eigenvalues)];
```
Comment (roystgnr) [inline src/gp/src/GPMSA.C]: I think this code is wrong, but I don't think this is the right fix here.  This is the lambda_eta term, isn't it?  So if num_svd_terms < num_nonzero_eigenvalues then we should be indexing with dimParameter; if not then we shouldn't be adding the inv(lambda_eta * K^T * K) term at all!

**Review by roystgnr — CHANGES_REQUESTED** [review]

File: src/gp/src/GPMSA.C (hunk 1 above)
Comment (dmcdougall) [reply src/gp/src/GPMSA.C]: Oh, I see.  This is the truncation error precision, not the emulator precision.  In that case I'll change the name of that variable as well as it's misleading.

Comment (roystgnr) [reply src/gp/src/GPMSA.C]: Please do, and sorry for the confusion.

**Review by roystgnr — APPROVED** [review]

**Comment by roystgnr** [comment]
:+1: when Travis is happy

**Pull request merged**